### ISSUE TITLE
[2.0.3] don't build

### ISSUE THREAD

**Issue by remicollet** [jedisct1/libsodium-php#138]
```
In file included from /usr/include/php/main/php.h:39:0,
                 from /dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:6:
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_auth_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2854:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_auth_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_auth_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2726:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_auth_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_generichash_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2865:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_generichash_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_generichash_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2737:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_generichash_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_kdf_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2876:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_kdf_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_kdf_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2748:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_kdf_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_secretbox_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2887:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_secretbox_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_secretbox_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2759:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_secretbox_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_shorthash_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2898:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_shorthash_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_shorthash_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2770:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_shorthash_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: error: redefinition of 'zif_sodium_crypto_stream_keygen'
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2909:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_stream_keygen)
 ^~~~~~~~~~~~
/usr/include/php/Zend/zend_API.h:66:23: note: previous definition of 'zif_sodium_crypto_stream_keygen' was here
 #define ZEND_FN(name) zif_##name
                       ^
/usr/include/php/Zend/zend_API.h:68:41: note: in definition of macro 'ZEND_NAMED_FUNCTION'
 #define ZEND_NAMED_FUNCTION(name)  void name(INTERNAL_FUNCTION_PARAMETERS)
                                         ^~~~
/usr/include/php/Zend/zend_API.h:69:52: note: in expansion of macro 'ZEND_FN'
 #define ZEND_FUNCTION(name)    ZEND_NAMED_FUNCTION(ZEND_FN(name))
                                                    ^~~~~~~
/usr/include/php/main/php.h:337:24: note: in expansion of macro 'ZEND_FUNCTION'
 #define PHP_FUNCTION   ZEND_FUNCTION
                        ^~~~~~~~~~~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:2781:1: note: in expansion of macro 'PHP_FUNCTION'
 PHP_FUNCTION(sodium_crypto_stream_keygen)
 ^~~~~~~~~~~~
In file included from /usr/include/php/main/php.h:39:0,
                 from /dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:6:
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:129:24: warning: 'AI_StateByReference' defined but not used [-Wunused-const-variable=]
 ZEND_BEGIN_ARG_INFO_EX(AI_StateByReference, 0, 0, 1)
                        ^
/usr/include/php/Zend/zend_API.h:118:38: note: in definition of macro 'ZEND_BEGIN_ARG_INFO_EX'
  static const zend_internal_arg_info name[] = { \
                                      ^~~~
/dev/shm/BUILD/php-pecl-libsodium2-2.0.3/NTS/libsodium.c:46:24: warning: 'AI_FourStrings' defined but not used [-Wunused-const-variable=]
 ZEND_BEGIN_ARG_INFO_EX(AI_FourStrings, 0, 0, 3)
                        ^
/usr/include/php/Zend/zend_API.h:118:38: note: in definition of macro 'ZEND_BEGIN_ARG_INFO_EX'
  static const zend_internal_arg_info name[] = { \
                                      ^~~~
Makefile:194: recipe for target 'libsodium.lo' failed
make[1]: *** [libsodium.lo] Error 1

```

**Comment by remicollet** [comment]
Same report by travis....

**Comment by remicollet** [comment]
Fixed in 2.0.4
